content/docs/integrations/nextjs.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Connect a Next.js application to Neon
+enableTableOfContents: true
+redirectFrom:
+  - /docs/quickstart/vercel
+---
+
+Next.js by Vercel is an open-source web development framework that enables React-based web applications. This topic describes how to create a Neon project and access it from a Next.js application.
+
+To create a Neon project and access it from a Next.js application:
+
+1. [Create a Neon project](#create-a-neon-project)
+2. [Create a Next.js project](#create-a-nextjs-project)
+3. [Add a PostgreSQL client to your app](#add-a-postgresql-client-to-your-app)
+4. [Add your Neon connection details](#add-your-neon-connection-details)
+5. [Connect to the Neon database](#connect-to-the-neon-database)
+
+## Create a Neon project
+
+To create a Neon project:
+
+1. Navigate to the [Projects](https://console.neon.tech/app/projects) page in the Neon Console.
+2. Click **New Project**.
+3. Enter a name for your project and click **Create Project**.
+4. After creating a project, you are directed to the project **Dashboard**, where a connection string with your password is provided under **Connection Details**. The connection string includes your password until you navigate away from the **Dashboard**. Copy the connection string. It is used later to connect to your Neon project.
+
+For additional information about creating projects, see [Setting up a project](/docs/get-started-with-neon/setting-up-a-project).
+
+## Create a Next.js project
+
+Create a Next.js project if you do not have one. For instructions, see [Create a Next.js App](https://nextjs.org/learn/basics/create-nextjs-app/setup), in the Vercel documentation.
+
+## Add a PostgreSQL client to your app
+
+Add a PostgreSQL client to your app, such as `Postgres.js`. For instructions, refer to the [postgres.js Getting started](https://www.npmjs.com/package/postgres).
```

Comment:
is there a template available in vercel that uses Postgres.js already? maybe we can refer to that as an example?